src/reporting/DisputeBondToken.sol

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+
+pragma solidity ^0.4.13;
+
+import 'ROOT/reporting/Branch.sol';
+import 'ROOT/reporting/ReputationToken.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/libraries/DelegationTarget.sol';
+import 'ROOT/libraries/Typed.sol';
+import 'ROOT/libraries/Initializable.sol';
+import 'ROOT/libraries/token/ERC20Basic.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+// CONSIDER: This could probably just be made Ownable instead if implementing ERC20Basic
+contract DisputeBondToken is DelegationTarget, Typed, Initializable, ERC20Basic {
+    using SafeMathUint256 for uint256;
+
+    IMarket private market;
+    address private bondHolder;
+    bytes32 private disputedPayoutDistributionHash;
+    uint256 private bondRemainingToBePaidOut;
+
+    function initialize(IMarket _market, address _bondHolder, uint256 _bondAmount, bytes32 _payoutDistributionHash) public beforeInitialized returns (bool) {
+        endInitialization();
+        market = _market;
+        bondHolder = _bondHolder;
+        disputedPayoutDistributionHash = _payoutDistributionHash;
+        bondRemainingToBePaidOut = _bondAmount * 2;
+        totalSupply = 1;
+        return true;
+    }
+
+    function withdraw() public returns (bool) {
+        require(msg.sender == bondHolder);
+        require(!market.isContainerForDisputeBondToken(this) || (market.isFinalized() && market.getFinalPayoutDistributionHash() != disputedPayoutDistributionHash));
+        require(getBranch().getForkingMarket() != market);
+        ReputationToken reputationToken = getReputationToken();
+        uint256 amountToTransfer = reputationToken.balanceOf(this);
+        bondRemainingToBePaidOut = bondRemainingToBePaidOut.sub(amountToTransfer);
+        reputationToken.transfer(bondHolder, amountToTransfer);
+        return true;
+    }
+
+    // FIXME: We should be minting coins in this scenario in order to achieve 2x
+    // target payout for bond holders during a fork.  Ideally, the amount minted is
+    // capped at the amount of tokens redeemed on other branches, so we may have to
+    // require the user to supply branches to deduct from with their call to this.
```

Comment:
Style: Don't put hard newlines in prose text.  Editor softwraps solve this problem well and hard newlines like this will totally screw up viewing the file/diff on a phone, side-by-side in a smallish browser window, etc.